tools/cli/main.c

```diff
@@ -262,19 +262,25 @@ TDNFCliShowVersion(
     PTDNF_CMD_ARGS pCmdArgs
     )
 {
+    uint32_t dwError = 0;
```

Comment:
why do we need dwError here at all?

Reply:
It's used in the `CHECK_JD_NULL/CHECK_JD_RC` macros.